client/src/objects/drawable.rs

```diff
@@ -58,6 +56,7 @@ impl<'lua> Drawable<'lua> {
 
     pub fn get_surface(&self) -> rlua::Result<Value<'lua>> {
         let drawable = self.state()?;
+        trace!("get_surface -> {}", drawable.surface.is_some());
```

Comment:
This log is unnecessary.

(Btw you might find the `dbg!` macro helpful in the future for debugging, it also makes it more clear that "this is logging that shouldn't actually be in the final solution")